_maps/MariosSwitchboard/mariosswitchboard.yaml

```diff
@@ -0,0 +1,194 @@
+---
+name:
+  en: Mario's Switchboard
+desc:
+  en: Switches propell you around the board. After 4 presses the direction reverses. The green !'s on the mini-map, move again squares, indictate the direction the switches will rotate, clockwise when near the top & counter-clockwise when near the bottom.  
+ruleSet: Standard
+theme: DragonQuest
+initialCash: 2800
+targetAmount: 20000
+baseSalary: 250
+salaryIncrement: 250
+maxDiceRoll: 6
+frbFiles:
+- marioswitch1
+- marioswitch2
+- marioswitch3
+- marioswitch4
+- marioswitch5
+- marioswitch6
+- marioswitch7
+- marioswitch8
+background: grid_green
+mapIcon: switchICON
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/mariosswitchboard.music.zip
+  - https://drive.google.com/file/d/1THXTKlOFLVaSuwIReoeVw0EVi0fHkLPf/view?usp=sharing

```

Comment:
The Google Drive download link for this one should be: `https://drive.google.com/uc?export=download&confirm=1&id=1THXTKlOFLVaSuwIReoeVw0EVi0fHkLPf`

Reply:
I'll hold off on fixing this one since it's going to be the new Star Carnival board.